pom.xml

```diff
@@ -51,6 +51,8 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
         <transformation.api.version>0.3.0-SNAPSHOT</transformation.api.version>
+        <commons-api.version>0.0.1-SNAPSHOT</commons-api.version>
+        <commons-impl.version>0.0.1-SNAPSHOT</commons-impl.version>
```

Comment:
We should just combine these to make `commons.version`, since these will always be the same.

Reply:
fixed